llm4pddl/utils.py

```diff
@@ -176,6 +176,33 @@ def get_custom_task(benchmark_name: str, task_num: int) -> Task:
     return Task(domain_file, problem_file)
 
 
+def prep_problem(problem: str) -> str:
+    """Maps a string of a problem file to a new string.
+
+    This new string has had all space removed between grounded atoms.
+    This is for use before querying llm in order to reduce tokens.
+    """
+    # Gets rid of leading and trailing spaces in every line:
+    new = [piece.strip() for piece in problem.split('\n')]
+    # Moves ')' characters to previous line.
```

Comment:
this is a good idea, but i wonder if we can write the code in a simpler way. would the following accomplish the same thing?
`new = new.replace("\n)", ")")`
or something like that?